src/server/routes/questions.js

```diff
@@ -29,22 +27,22 @@ router.get(
 );
 
 router.get(
-  '/:id',
+  '/:slug',
```

Comment:
my current pr uses this as an id pretty heavily

Reply:
Ok, let me change that!